packages/pwa-buildpack/lib/Utilities/graphQL.js

```diff
@@ -1,3 +1,5 @@
+const debug = require('../util/debug').makeFileLogger(__filename);
```

Comment:
do we want to keep all these debug statements in this file too?

Reply:
It's a file logger object definition.